baserun/instrumentation/instrumented_wrapper.py

```diff
@@ -152,7 +153,7 @@ def instrumented_function(*args, **kwargs):
                     raise e
 
                 # If this is a streaming response, wrap it, so we can capture each chunk
-                if isinstance(response, GeneratorType):
+                if isinstance(response, Iterator):
```

Comment:
Anthropic's response is not a generator, updated this and above to check if it is an Iterator / AsyncIterator